### ISSUE TITLE
ScalaPB 0.11.13 missing release note

### ISSUE THREAD

**Issue by eed3si9n** [scalapb/ScalaPB#1575]
## steps

https://github.com/scalapb/ScalaPB/releases/tag/v0.11.13

## problem

The release note is missing any details.

## expectations

Please generate a release note or link to https://github.com/scalapb/ScalaPB/compare/v0.11.12...v0.11.13

**Comment by thesamet** [comment]
Hi @eed3si9n , trying to understand the intent behind the issue, as many of the previous releases do not have release notes - and this hasn't been requested before. I do keep note of the important changes in https://github.com/scalapb/ScalaPB/blob/master/CHANGELOG.md

**Comment by eed3si9n** [comment]
I guess I only saw https://github.com/scalapb/ScalaPB/releases/tag/v0.11.12, and assumed that 0.11.13 was somehow missing the release note. Feel free to close this issue, if v0.11.12 was more of an anomaly.
Specifically, I was wondering when in history `scalapb.Message` went away, since I recently ran into

```
Cause: java.lang.ClassNotFoundException: scalapb.Message
```

I guess someone is using ScalaPB 0.9.0?
